extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -217,4 +217,11 @@ public class JibConfig {
      */
     @ConfigItem(defaultValue = "true")
     public boolean useCurrentTimestamp;
+
+    /**
+     * Whether to set the modification time (last modified time) of the files put by Jib in the image to the actual
+     * build time. Otherwise, the modification time will be set to the Unix epoch (00:00:00, January 1st, 1970 in UTC).
```

Comment:
I think we should also explain what this does, basically what the effect of turning it off means for layering.

Reply:
Appending something like the paragraph below?

_If the modification time is constant (flag is set to false so Unix epoch is used) across two consecutive builds, the docker layer sha256 digest will be different only if the actual files added by Jib to the docker layer were changed. More exactly, having 2 consecutive builds will generate different docker layers only if the actual content of the files within the docker layer was changed._

Reply:
👍🏼